Sources/SwifterSwift/Foundation/DateExtensions.swift

```diff
@@ -487,6 +487,20 @@ public extension Date {
         return timeIntervalSince1970
     }
 
+    /// SwifterSwift: Current day in a year
+    var dayInYear: Int {
+        return calendar.ordinality(of: .day, in: .year, for: self)!
+    }
+
+    /// SwifterSwift: How many days in a year
+    var numberOfDaysInYear: Int {
+        return (calendar.range(of: .day, in: .year, for: self)!.upperBound) - 1
+    }
+
+    /// SwifterSwift: How many days are left in a year
+    var numberOfDaysLeftInYear: Int {
```

Comment:
What about `remainingDays(in: .year)`, that would be a more general extension I think :))

Reply:
That's a great idea! Thanks so much!
So this method will be useful for both month and year, right ?

Reply:
Yeap :)

Reply:
But then you'd need it to return an optional, because if the caller was `remainingDays(in: .seconds)` you would get back `nil`.

Reply:
This is picky but by Apple's naming conventions this should _probably_ be `daysRemaining(in:)`

Reply:
@chulbert That makes sense :)